### ISSUE TITLE
Add an output option

### ISSUE THREAD

**Issue by tivac** [developit/esbench#4]
Right now to share the results of tests requires weird copy/paste or a screenshot. Would be nice if there was an "Export" button that would copy a markdown/plain text/csv/something variant to your clipboard for easier reporting of test results.
#3 would make this less important, but still useful.

**Comment by developit** [comment]
Agreed 100%! I'm more excited about #3, but building the storage bit would make this export option basically a one-liner so it's an easy feature to add.
